manifest.json

```diff
@@ -3,7 +3,10 @@
     "name": "Casey's Extension",
     "version": "1.0.0",
     "description": "Hello World!",
-    "permissions": ["webRequest" , "http://*/*", "https://*/*", "webRequestBlocking"],
+    "permissions": ["webRequest",
+        "http://*/*",
+        "https://*/*",
+        "webRequestBlocking"],
```

Comment:
Looking good, just one other little thing to bring the formatting completely in line with best practices. Compare to what we have in the extension manifest & you'll see what I mean ~